Classes/Event/EventBus.php

```diff
@@ -1,54 +0,0 @@
-<?php
-namespace Neos\Cqrs\Event;
-
-/*
- * This file is part of the Neos.Cqrs package.
- *
- * (c) Contributors of the Neos Project - www.neos.io
- *
- * This package is Open Source Software. For the full copyright and license
- * information, please view the LICENSE file which was distributed with this
- * source code.
- */
-
-use Neos\Cqrs\Event\Exception\EventBusException;
-use Neos\Cqrs\EventListener\EventListenerLocator;
-use TYPO3\Flow\Annotations as Flow;
-use TYPO3\Flow\Log\SystemLoggerInterface;
-use TYPO3\Flow\Utility\TypeHandling;
-
-/**
- * @Flow\Scope("singleton")
- */
-class EventBus
```

Comment:
I'm a bit against removing too much of the common concept or vocabulary around CQRS ... and an EventBus ... if our EventPublisher make the event bus "obsolete", let's rename the EventPublisher ... no ?